docs/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# ARBITRARY DOCS CHANGE
```

Comment:
cc @alyssawilk - testing with a docs only chnage - ill try mobile after

Reply:
hmm, so first problem i see is that `check` stage is required for pr to land - i dont think we can make that conditional - i guess we could supress the individual jobs in the `check` stage

Reply:
doesnt seem to have worked anyway - wierd - i will debug

Reply:
arrg - it has `--quiet` which will not work - ill PR to fix that